src/ansys/fluent/core/codegen/datamodelgen.py

```diff
@@ -362,12 +370,42 @@ def _delete_generated_files(self):
             shutil.rmtree(Path(_SOLVER_DM_DOC_DIR))
 
 
-def generate(version, sessions: dict):
+def generate(version, static_infos: dict):
     """Generate datamodel API classes."""
-    return DataModelGenerator(version, sessions).write_static_info()
+    return DataModelGenerator(version, static_infos).write_static_info()
 
 
 if __name__ == "__main__":
```

Comment:
We support running individual *gen files: `python src/ansys/fluent/core/codegen/datamodelgen.py`